config/v3_0/types/storage_test.go

```diff
@@ -148,3 +148,105 @@ func TestStorageValidate(t *testing.T) {
 		}
 	}
 }
+
+func TestStorageValidateWarnings(t *testing.T) {
+	tests := []struct {
+		in  Storage
+		at  path.ContextPath
+		out error
+	}{
+		{
```

Comment:
Let's maybe add a comment on each of these explaining the case they're testing?

Reply:
Yeah, I agree, and will get to that shortly.